### PR TITLE
#85 - SQLServer Native Backup and Restore on RDS

### PR DESCRIPTION
**Pull request by JAG-010**
I have created a document related to database, hope it will be helpful.

### CODE REVIEW

File: Projects/DBS/DBS03/DBS03-AWS201.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+- Included in the Free Tier
+
+
+## Estimated time to complete
+- 45-60 minutes
+
+
+## Tips
+- Having database knowledge is an added advantage
+- 750 hours of Amazon RDS Single-AZ `db.t2.micro` included in free tier
```
Comment (what-name) [inline Projects/DBS/DBS03/DBS03-AWS201.md]: This should be placed in the "Costs" section please

Comment (JAG-010) [reply Projects/DBS/DBS03/DBS03-AWS201.md]: You want me to move this line to "Costs" section?
> 750 hours of Amazon RDS Single-AZ `db.t2.micro` included in free tier

Please excuse me, am new to github

**Review by JAG-010 — COMMENTED** [review]
I have updated the costs section

**Comment by madebygps** [comment]
Fantastic! thank you for your contribution.

**Pull request merged**